src/popover/container.tsx

```diff
@@ -30,7 +30,9 @@ export interface PopoverContainerProps {
   arrow: (position: InternalPosition | null) => React.ReactNode;
   children: React.ReactNode;
   renderWithPortal?: boolean;
-  size: PopoverProps.Size;
+  // The "content" size is a special one that does not have the associated container-body-size-{size} class.
+  // It can be used to create larger popovers where the responsiveness is handled by the content.
```

Comment:
I do not think this is something that needs explanation. Not every `variant-${variantName}` class in any other component has this comment. 

Leaving this comment you add more confusion than clarity.

Reply:
removed